### DISCUSSION TITLE
Copying files... prompt can be made better.

### DISCUSSION THREAD

**Discussion by thevipinmishra** [withastro/roadmap#375] [Proposal]
I was bootstraping a new astro project and was stuck on this step for a long time.

![image](https://user-images.githubusercontent.com/52159805/200110069-53685fee-880f-45db-b1e1-267e98dafb1c.png)

I had no idea if my internet was slow or anything, a progress bar (with internet speed, done/remaining meter) can be there for a better UX.

Just a thought!

**Comment by florian-lefebvre** [comment]
Closing because this is outdated
